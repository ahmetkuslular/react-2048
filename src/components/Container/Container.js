import React from 'react'
import styled from 'styled-components'

function Container({children}) {
  return (
      <Main>
        <Content>{children}</Content>
      </Main>
  )
}

const Main = styled.main`
  align-items: center;
  display: flex;
  height: 100%;
  justify-content: center;
  width: 100%;
  
`;

const Content = styled.div`
  width: 420px;
  display: flex;
  flex-direction: column;
`

export default Container
