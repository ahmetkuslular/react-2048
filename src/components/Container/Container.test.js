import React from 'react';
import ReactDOM from 'react-dom';
import { render } from 'enzyme';
import toJson from 'enzyme-to-json';

import Container from './Container';

describe('components/Container', () => {
  it('renders without crashing', () => {
    const div = document.createElement('div');
    ReactDOM.render(<Container />, div);
  });

  it('renders markup correctly', () => {
    const tree = toJson(render(<Container>Children</Container>));
    expect(tree).toMatchSnapshot();
  });
});
