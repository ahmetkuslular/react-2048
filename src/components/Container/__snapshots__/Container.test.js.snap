// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components/Container renders markup correctly 1`] = `
<main
  class="sc-bdVaJa ihPaUS"
>
  <div
    class="sc-bwzfXH gnVKwi"
  >
    Children
  </div>
</main>
`;
